ontology/doid.md

```diff
@@ -43,6 +43,13 @@ license:
   url: https://creativecommons.org/publicdomain/zero/1.0/
   label: CC0 1.0 Universal
 usages:
+  - user: https://disease-ontology.org/community/collaborators
```

Comment:
This is not a real user - for the usages section, we look for named users like iedb and alliance, see comments below.